packages/e2e-tests/src/specs/wordpress/webStoriesBlock.js

```diff
@@ -38,7 +39,13 @@ const EMBED_BLOCK_CONTENT = `
 <!-- /wp:web-stories/embed -->
 `;
 
-describe('Web Stories Block', () => {
+const maybe = !checkVersion(process.env?.WP_VERSION, '5.8')
```

Comment:
That `checkVersion` util seems a bit overkill. Can't we just do `process.env.WP_VERSION.startsWith('5.8')`?

Reply:
I was trying to copy PHP's [version_compare](https://www.php.net/manual/en/function.version-compare.php), where it would allow use filter by point releases, which was an issue [here](https://github.com/google/web-stories-wp/pull/5550). We had to lock to 5.3.4 as there were issues with 5.3.x.